combat/Combat.gd

```diff
@@ -42,15 +42,19 @@ func _ready():
 	#for i in range(1):
 	for i in range(MAX_ENEMIES):
 		var position = enemy_positions[i]
-		var creature = CombatCreature.CombatCreature.new("Monster" + str(i), EnemyScene.instance(), CombatGlobal.CreatureSize.MEDIUM, position, 50, 50, CombatCreature.Stats.new(1, 2, (i + 1), 1, 1))
+		var creature = CombatCreature.new("Monster" + str(i), EnemyScene.instance(), CombatGlobal.CreatureSize.MEDIUM, position, 50, 50, CombatCreature.Stats.new(1, 2, (i + 1), 1, 1))
```

Comment:
More magic numbers, stat block should be defined somewhere

Reply:
This will be completely changed once creatures are added. This is just placeholder code.

I created an issue for this: [Combat - Add Creatures](https://github.com/demwilson/gameoff2020/issues/42)

Reply:
Acceptable